simple_salesforce/api.py

```diff
@@ -141,6 +151,14 @@ def __init__(
         self.apex_url = ('https://{instance}/services/apexrest/'
                          .format(instance=self.sf_instance))
 
+        # Replace default formats with the custom URL
+        if self.custom_url:
+            base_url_root = custom_url + '/services/data/v{version}/'
```

Comment:
Please use `.format()` rather than concatenation for consistency (I also wouldn't be mad at interpolation)